app/presenters/publishing_api/worldwide_organisation_presenter.rb

```diff
@@ -272,6 +272,7 @@ def details
         world_location_names:,
```

Comment:
Happy with the code here, it'd be good if you could add a bit to the commit message to explain why it's written this way, basically just a note on why we ignore the about page and how these pages work.